guides/source/caching_with_rails.md

```diff
@@ -512,6 +512,26 @@ class ProductsController < ApplicationController
 end
 ```
 
+Sometimes we want to cache response (static page) that never gets expired.
+To achieve that, we can use `http_cache_forever` helper and by doing so browser
+and proxies will cache it indefinitely.
+
+By default cached response will be private. To make it public response, pass
+`public: true` parameter to this method. In this helper, `last_modified` header
```

Comment:
We can replicate this documentation from the method- 
`To allow proxies to cache the response, set +true+ to indicate that they can serve the cached response to all users.`